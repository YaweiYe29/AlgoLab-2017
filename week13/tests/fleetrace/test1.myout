1217
1208
1432
1425
1441
1450
1436
1428
1444
