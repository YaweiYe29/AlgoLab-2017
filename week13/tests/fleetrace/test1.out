1217
1208
1432
1425
1441
1450
1436
1428
1444
1434
1433
1434
1459
1453
1426
1441
1430
1431
1448
1433
1449
1454
1419
1442
1437
1439
1421
1424
1439
1416
